app/app.module.ts

```diff
@@ -19,8 +19,10 @@ import { HelpComponent } from './help/help.component';
 import { CoreModule } from './core/core.module';
 import { FooterComponent } from './footer/footer.component';
 import { AdminModule } from './admin/admin.module';
+import { SharedModule } from './shared/shared.module';
 import { EqualValidatorDirective } from './authentication/signup/equal-validator.directive';
 import { ManageUserComponent } from '../app/userprofile/userprofile.component';
```

Comment:
shared module is for upload-picture.component. I cant remove it. I will remove the unused upload-picture component as per your next comment